cypress/integration/customizer/hfg/hfg-logo-site-identity.spec.js

```diff
@@ -0,0 +1,172 @@
+const color = '#285078';
+const rgbColor = 'rgb(40, 80, 120)';
+
+describe('Header Builder Logo & Site Identity settings', function () {
+	it('Sets changes and checks them', function () {
+		cy.login();
+		aliasRestRoutes();
+		hideTitleAndTagline();
+		cy.wait('@customizerSave').then((req) => {
+			expect(req.response.body.success).to.be.true;
+			expect(req.status).to.equal(200);
+			cy.wait(2000);
+			checkFrontEndHided();
+		});
+		setupCustomizer();
+		cy.wait('@customizerSave').then((req) => {
+			expect(req.response.body.success).to.be.true;
+			expect(req.status).to.equal(200);
+			cy.wait(2000);
+			checkFrontEnd();
+		});
+
+	});
+});
+
+function aliasRestRoutes() {
+	let home = Cypress.config().baseUrl;
+	cy.server().
+	route('POST', home + '/wp-admin/admin-ajax.php').
+	as('customizerSave');
+}
+
+function hideTitleAndTagline() {
+	cy.visit('/wp-admin/customize.php');
+
+	cy.get('#accordion-panel-hfg_header').should('be.visible').
+	click();
+
+	//Switches to Logo & Site Identity settings.
+	cy.get(
+		'.hfg--builder-show .hfg--panel-desktop .hfg--row-main .grid-stack-item[title="Logo & Site Identity"]').
+	should('be.visible').
+	click();
+
+	//Hides the site Title and Tagline
+	cy.get(
+		'#inspector-toggle-control-0').
+	click();
+	cy.get(
+		'#inspector-toggle-control-1').
+	click();
+	cy.get('#save').click();
+}
```

Comment:
This function could be a Cypress command. Check the doc of Custom Commands: docs.cypress.io/api/cypress-api/custom-commands.html